docs/source/getting_started/active_learning.rst

```diff
@@ -1,22 +1,26 @@
 .. _lightly-active-learning:
 
-Active Learning
+Active-Learning
```

Comment:
Are you sure about the hyphen? I've never seen that anywhere before.

Reply:
Hmm, I thought it was with hyphen :S
Now that I checked everywhere, you're right. Will undo